### DISCUSSION TITLE
Can't find Docker after installation of AI plugin

### DISCUSSION THREAD

**Discussion by 23f1002974** [Acly/krita-ai-diffusion#942] [General]
Hi,

I am trying to install AI Diffusion of Krita, after installation I have to enable Docker according to installation [guide](https://www.interstice.cloud/plugin)
but it doesn't show that option.
![image](https://github.com/user-attachments/assets/abbcc853-323f-4983-a85b-ff8ca851875b)
even though it is already installed
![image](https://github.com/user-attachments/assets/823aa173-e417-44de-ab45-123c9b18eed3)

Please help, It'll be great help.
Thanks

**Comment by minsky91** [comment]
same here

**Reply by minsky91** [reply]
It worked, but only after resorting to manually replacing of the old Krita AI plugin folder contents with the new one

**Comment by kings952** [comment]
have the same  issue but when i see de phyton plugins show me this 
![image](https://github.com/user-attachments/assets/e41830f7-12a6-4c81-bbbf-2d3096bf1b92)

**Comment by White-Flare** [comment]
Same problem. Tried going back to a previous version of Krita (5.2.3 -> 5.2.1). I ended up having to install the 1.19 version of this plugin vs the new 1.20.1 to get the docker to appear.

**Comment by 23f1002974** [comment]
Well,
It started working well after **fresh installation** of _latest_ **Krita 5.2.3** and plugin installation.
